tree/ntupleutil/v7/test/ntuple_inspector.cxx

```diff
@@ -277,21 +270,26 @@ TEST(RNTupleInspector, ColumnSize)
    std::unique_ptr<TFile> file(TFile::Open(fileGuard.GetPath().c_str()));
    auto ntuple = file->Get<RNTuple>("ntuple");
    auto inspector = RNTupleInspector::Create(ntuple);
-   auto reader = RNTupleReader::Open(ntuple);
 
-   for (std::size_t i = 0; i < reader->GetDescriptor()->GetNLogicalColumns(); ++i) {
-      EXPECT_GT(inspector->GetOnDiskColumnSize(i), 0);
-      EXPECT_GT(inspector->GetInMemoryColumnSize(i), 0);
-      EXPECT_LT(inspector->GetOnDiskColumnSize(i), inspector->GetInMemoryColumnSize(i));
+   std::uint64_t totalOnDiskSize = 0;
+
+   for (std::size_t i = 0; i < inspector->GetDescriptor()->GetNLogicalColumns(); ++i) {
+      auto colInfo = inspector->GetColumnInfo(i);
+      totalOnDiskSize += colInfo.GetOnDiskSize();
+
+      EXPECT_GT(colInfo.GetOnDiskSize(), 0);
+      EXPECT_GT(colInfo.GetInMemorySize(), 0);
+      EXPECT_LT(colInfo.GetOnDiskSize(), colInfo.GetInMemorySize());
```

Comment:
I would also suggest adding a test in which we do not use compression.  Given that we can also set a specific encoding for the columns, we can be more specific in these comparisons, which is useful to actually test that we are computing the results correctly.

The same applies to the `FieldInfo` test below.